test/tests/functions/string.json

```
@@ -44,6 +44,16 @@
           "type": "unresolved-variable"
         }
       ]
+    },
+    {
+      "description": "NFC: keys are normalized",
+      "src": ".local $x = {\u1E0A\u0323 :string} .match $x \u1E0A\u0323 {{Not normalized}} \u1E0C\u0307 {{Normalized}} * {{Wrong}}",
+      "expErrors": [{"type": "duplicate-variant"}]
+    },
+    {
+      "description": "NFC: keys are normalized",
+      "src": ".local $x = {\u1E0A\u0323 :string} .match $x \u1E0C\u0307 {{Right}} * {{Wrong}}",
+      "exp": "Right"
     }
```

We don't actually require this (matching keys to a non-normalized compare value). We should, but we don't. I think we need to update `:string` selection to also normalize the comparison value. Especially as the reverse works:
```suggestion
    {
      "description": "NFC: keys are normalized",
      "src": ".local $x = {\u1E0C\u0307 :string} .match $x \u1E0A\u0323 {{Right}} * {{Wrong}}",
      "exp": "Right"
    }
```

---

Added in [6bcd592](https://github.com/unicode-org/message-format-wg/pull/904/commits/6bcd5922e660244e2ce574d89375f99e8b15e0a6)